_posts/en/newsletters/2024-05-08-newsletter.md

```diff
@@ -0,0 +1,311 @@
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Bitcoin Inquisition][bitcoin inquisition repo], and [BINANAs][binana
+repo]._
+
+- [Bitcoin Core #28970][] and [#30012][bitcoin core #30012] add support
+  for a limited form of [package relay][topic package relay] that
+  doesn't require any changes to the P2P protocol.  Imagine Alice has a
+  parent transaction below any of her peer's [BIP133][] feefilter
+  settings, discouraging her from relaying it out of the knowledge that
+  none of her peers will accept it.  She also has a child transaction
+  that pays enough feerate alongside its parent for both of them to be
+  above the feefilter.  Alice and her peer perform the following
+  process:
+
+  - Alice relays the child transaction to her peer.
+
+  - Her peer realizes it doesn't have the transaction's parent, so it
+    puts the transaction into its _orphan pool_.  All versions of
+    Bitcoin Core for over a decade have had an orphan pool where they
+    temporarily store a limited number of transactions that were
+    received before their parents.  This compensates for the fact that,
+    on a P2P network, transactions can sometimes naturally be received
+    out of order.
+
+  - A few moments later, Alice relays the parent transaction to her
+    peer.
+
+  - Before this merged PR, the peer would notice that the parent's
+    feerate was too low and refuse to accept it; now that it had
+    evaluated the parent transaction, it would also remove
+    the child transaction from the orphan pool.  After this PR, the peer
+    notices that it has a child for the parent in its orphan pool and
+    evaluates aggregate feerate of both transactions together, allowing
+    them both into the mempool if that feerate is above its floor (and
+    if they're both otherwise acceptable according to the node's local
+    policy).
+
+  It's known that this mechanism can be defeated by an attacker.
+  Bitcoin Core's orphan pool is a circular buffer that can be added to
+  by all of its peers, so an attacker who want to prevent this type of
+  package relay can spam peers with many orphan transactions,
+  potentially leading to a fee-paying child transaction being evicted
+  before it's parent is received.  A [follow-up PR][bitcoin core #27742]
+  may give each peer exclusive access to part of the orphan pool to
+  eliminate this concern.  Additional improvements requiring changes to
+  the P2P protocol are described in [BIP331][].
+
+- [Bitcoin Core #28016][] begins waiting for all seed nodes to be polled
+  before polling DNS seeds.  Users can configure both seed nodes and DNS
+  seeds.  A seed node is a regular Bitcoin full node; Bitcoin Core can
```

Comment:
IIRC, It might be worth distinguishing in the writeup that DNS seed are hard coded and no seed nodes are hard coded. A point you somewhat touch on in the last sentence.